code/modules/admin/admin_verbs.dm

```diff
@@ -825,7 +835,17 @@ var/list/admin_verbs_proccall = list (
 		return
 
 	if(!istype(H))
-		return
+		if(istype(H, /mob/living/carbon/brain))

```

Comment:
Same as above. Only added out of necessity, it wouldn't work any other way and I tried to make it such that it wouldn't permit any un-mmi'd brains without IPC bodies from doing this.